atuin-client/src/database.rs

```diff
@@ -38,18 +42,32 @@ pub struct OptFilters {
     pub reverse: bool,
 }
 
-pub fn current_context() -> Context {
+pub fn current_context(settings: &Settings) -> Context {
     let Ok(session) = env::var("ATUIN_SESSION") else {
         eprintln!("ERROR: Failed to find $ATUIN_SESSION in the environment. Check that you have correctly set up your shell.");
         std::process::exit(1);
     };
     let hostname = format!("{}:{}", whoami::hostname(), whoami::username());
     let cwd = utils::get_current_dir();
+    let interpreter = env::var("ATUIN_INTERPRETER").ok();
```

Comment:
I've decided to pass interpreter through environment variable, not as the parameter. This felt more natural, as interpreter is used in `atuin history start` and `atuin history search` now, and may be will be in other commmands